sdl_viewer/src/node_drawer.rs

```diff
@@ -110,13 +110,22 @@ impl NodeDrawer {
 
             self.program.gl.Uniform1f(
```

Comment:
Could you add a TODO about the bug we just discussed?

Reply:
Done.